docs/api-docs/webhooks/webhook-events.mdx

```diff
@@ -928,7 +994,7 @@ Consult the [sites section of the Channel Webhooks Guide](/docs/integrations/web
 |:-------------------------|:------------|
 | store/sku/* | Fires for all store/sku events. |
 | store/sku/created | Fires when a new SKU is created.|
-| store/sku/updated | Fires when a SKU is updated. |
+| store/sku/updated | Fires when you edit a variant globally or for a channel locale as overrides. The `context` fields are present for only override updates. |
```

Comment:
I got this scope from the [google doc](https://docs.google.com/document/d/1uTeDIkd33EtmQrshLtp0Yh3Qows6OFj5PSZvNmP8n0k/edit). I'm not sure how it's different from `store/options/updated`. Should this actually be documented?

Reply:
@kristinapototska @bc-andreadao what are the plans for publishing this doc? For now we are missing `store/sku/updated` triggering for overrides at all and it won't be implemented in nearest time.

Reply:
As for `store/options/updated` and `store/sku/updated` - these two webhooks are different, one for options and another one for variants

Reply:
I'm not entirely sure when we want the doc published. From what you just said, I can remove `store/sku/updated` though, if that works for @kristinapototska

Reply:
@bc-andreadao, we want this document published as soon as we can. 
We moved incomplete scope (Product URLs) to a separate tiny project and this one (PROJECT-5655) is waiting for DevDocs only to be completed.